packages/unimodules-test-core/unimodule.json

```diff
@@ -0,0 +1,4 @@
+{
+  "name": "unimodules-test-core",
+  "platforms": ["ios", "android"]
+}
```

Comment:
Do we want this file to be treated as `unimodule`? I don't think so, as it communicates the wrong idea here.
The only benefit that I expect from having this file in this module is auto-adding to Android/iOS project 🤔 And this one could be done manually in Podfile/Gradle files for this library as it's the very special case 😉

Reply:
In general I agree.

My concern was that I don't really know of all the places where existence of this file might be used. However, if your intuition is we can safely get rid of it, I'll do it.

Reply:
AFAIK, this file is only used by our unimodules auto-installation script and communicated that the module is actually an unimodule (whatever that could mean right now), so please do.

Reply:
My second thought is: we don't want any script to assume it is an unimodule. So this file might be misleading. I've deleted it, thanks for this catch.